src/robot/libraries/dialogs_ipy.py

```diff
@@ -12,32 +12,183 @@
 #  See the License for the specific language governing permissions and
 #  limitations under the License.
 
+import wpf
+from System.Windows import Application, GridLength, SizeToContent, TextWrapping, \
+                           Thickness, Window, WindowStartupLocation, WindowState
+from System.Windows.Controls import Button, ColumnDefinition, Grid, Label, ListBox, \
+                                    PasswordBox, RowDefinition, TextBlock, TextBox
```

Comment:
We've used parenthesis to cut long import lines like

```python
from System.Windows import (Application, GridLength, SizeToContent, TextWrapping,
                            Thickness, Window, WindowStartupLocation, WindowState)
```

but that's definitely not a big issue.